api/src/home/analysis-platform/job_handlers/local_container/__init__.py

```diff
@@ -36,6 +36,8 @@ def start(self) -> str:
         try:
             reference_target: str = self.job.entity.get("referenceTarget", None)
             runner_entity: dict = self.job.entity["runner"]
+            if not runner_entity["image"].get("registryName", None):
+                raise KeyError("Container image in job runner")
```

Comment:
Also no need for this check. A KeyError will be raised on line 42 if value is not set